elections/tac-and-scir-election-process.md

```diff
@@ -68,7 +68,7 @@ Questions related to the nomination and/or election process should be directed t
 
 ## Appointment Process
 
-The Governing Board will appoint three people from the list of TAC and SCIR Nominees excluding those who got elected. The Governing Board will be provided with the results of the election from the community so that they can ensure appropriate diversity of the TAC during their vote.
+The Governing Board will appoint three people pulling from the list of TAC and SCIR Nominees excluding those who got elected and those they nominate. The Governing Board will be provided with the results of the election from the community so that they can ensure appropriate diversity of the TAC during their vote.
```

Comment:
```suggestion
The Governing Board will appoint three (3) seats on the Technical Advisory Council. These individuals will be selected from the following nominee categories:

- TAC and SCIR nominees for the current election who were _not_ elected
- individuals from [OpenSSF member organizations](https://openssf.org/about/members/) that are nominated by Governing Board representatives 

The Governing Board will be provided with the results of the election from the community so that they can ensure appropriate diversity of the TAC during their vote.
```

Made some adjustments here so that the exclusion language reads more clearly.

Reply:
> ensure appropriate diversity

What does this mean more precisely?